Annanya/Question2/Ans2_String_RE.py

```diff
@@ -0,0 +1,7 @@
+import re
+str=input('Enter the string')
+rhy=re.search('.ake',str)
```

Comment:
This program will match any word starting with any letter but contains 'ake' later on. In the question you specified that it will match any string containing 'cake' not '.ake'